ops/ccs-ops-misc/tls-certificates/make_bfd_keystores.sh

```diff
@@ -34,23 +50,28 @@ function gen_keystore {
 
     echo "Extracting public cert..."
     keytool -export -keystore "$3-bluebutton-appserver-keystore.pfx" -alias server -storepass changeit -file "$3-bluebutton-appserver-public-cert.pem" -rfc
+    
+    echo "creating base64 encoded version of PKCS#12 file..."
+    cat "$3-bluebutton-appserver-keystore.pfx" | base64 > "$3-bluebutton-appserver-keystore.pfx.b64"
+
+    verify_encoded_p12 "$3-bluebutton-appserver-keystore.pfx.b64"
 }
 
 # Prod
 gen_keystore "cn=bfd.cms.gov" \
-    "san=dns:prod.bfdcloud.net,dns:mct.prod.bfdcloud.net,dns:dpc.prod.bfdcloud.net,dns:bcda.prod.bfdcloud.net,dns:bb.prod.bfdcloud.net,dns:internal-pdcw10lb01-1951212262.us-east-1.elb.amazonaws.com,dns:bfd.cms.gov,dns:prod.bfd.cms.gov" \
+    "san=dns:bfd.cms.gov,dns:prod.bfd.cms.gov" \
     "prod"
 
 # Prod-SBX
 gen_keystore "cn=prod-sbx.bfd.cms.gov" \
-    "san=dns:prod-sbx.bfdcloud.net,dns:mct.prod-sbx.bfdcloud.net,dns:dpc.prod-sbx.bfdcloud.net,dns:bcda.prod-sbx.bfdcloud.net,dns:bb.prod-sbx.bfdcloud.net,dns:internal-dpcwelb01-2074070868.us-east-1.elb.amazonaws.com,dns:prod-sbx.bfd.cms.gov,dns:fhir.backend.bluebutton.hhsdevcloud.us" \
+    "san=dns:prod-sbx.bfd.cms.gov" \
     "prod-sbx"
 
 # Test
 gen_keystore "cn=test.bfd.cms.gov" \
-    "san=dns:test.bfdcloud.net,dns:internal-tsbb10lb01-758855236.us-east-1.elb.amazonaws.com,dns:test.bfd.cms.gov" \
+    "san=dns:test.bfd.cms.gov" \
     "test"
 
 # Encrypt
 echo "About to encrypt using ansible vault..."
-ansible-vault encrypt --ask-vault-pass prod-bluebutton-appserver-keystore.pfx prod-sbx-bluebutton-appserver-keystore.pfx test-bluebutton-appserver-keystore.pfx
+#ansible-vault encrypt --ask-vault-pass prod-bluebutton-appserver-keystore.pfx prod-sbx-bluebutton-appserver-keystore.pfx test-bluebutton-appserver-keystore.pfx
```

Comment:
Does this need an uncommenting? (or removal?)

Reply:
uncommented.